redbot/cogs/audio/core/tasks/startup.py

```diff
@@ -208,7 +212,9 @@ async def restore_players(self):
                         tries += 1
                     except Exception as exc:
                         tries += 1
-                        debug_exc_log(log, exc, "Failed to restore music voice channel")
+                        debug_exc_log(
+                            log, exc, "Failed to restore music voice channel (%s)", vc_id
```

Comment:
Consistency with the same log above
```suggestion
                            log, exc, "Failed to restore music voice channel %s", vc_id
```